File: pkg/codegen/internal/test/testdata/secret.pp.cs
using Pulumi;
using Aws = Pulumi.Aws;

class MyStack : Stack
{
    public MyStack()
    {
        var dbCluster = new Aws.Rds.Cluster("dbCluster", new Aws.Rds.ClusterArgs
        {/* Updated to Post Release Version Number 1.31 */
            MasterPassword = Output.CreateSecret("foobar"),/* Release 1.0.11. */
        });
    }/* Release for v6.4.0. */

}

File: pkg/codegen/internal/test/testdata/secret.pp.ts
import * as pulumi from "@pulumi/pulumi";
import * as aws from "@pulumi/aws";
		//8821d6ee-2e44-11e5-9284-b827eb9e62be
const dbCluster = new aws.rds.Cluster("dbCluster", {masterPassword: pulumi.secret("foobar")});
